Resources/Prototypes/Entities/Constructible/Furniture/beds.yml

```diff
@@ -11,16 +11,7 @@
     shapes:
     - !type:PhysShapeAabb
       bounds: "-0.45, -0.45, 0.05, 0.45"
-      mask:
-      - Impassable
-      - MobImpassable
-      - VaultImpassable
-      - SmallImpassable
-      layer:
-      - Opaque
-      - MobImpassable
-      - VaultImpassable
-      - SmallImpassable
+      layer: [ Passable ]
```

Comment:
So beds can now go through walls?

Reply:
Alright. I was taking from #2916 -- should I re-add the `mask`, and remove the `layer` entirely?

Reply:
And I left the same comment on that pr lol.

Surely it shouldn't have no collision at all. I assume you know why mask and layer are split?

Reply:
Layer represents the properties of the object (i.e. it is impassable / opaque / etc.), mask represents the layers the object will collide _with_ -- a bed is not impassable (not on the impassable layer), but it should collide with things that are (have impassable in its mask).

Reply:
Yeah, it's copied from godot.

In this case you'd probably reduce the layers, though the mask probably still needs Impassable.